index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/css?family=Poppins:100,100italic,200,200italic,300,300italic,regular,italic,500,500italic,600,600italic,700,700italic,800,800italic,900,900italic" rel="stylesheet" />
+    <script src="https://kit.fontawesome.com/2df075da03.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+  <header class="nav">
+    <nav>
+      <a href="#">My Logo</a>
```

Comment:
You must stick to the design as much as possible (e.g. font, colors, images, tests, margins) using [the templates ](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) in Figma. 
Your logo 
![image](https://user-images.githubusercontent.com/60292761/148406082-8fa31d00-a711-44f6-9c10-7ffe9d8c89c0.png)
Figma logo
![image](https://user-images.githubusercontent.com/60292761/148406154-fb0846aa-f89b-4297-9796-61153cc73bfb.png)
Please design the `a` tag in your style.css file